aiven/client/cli.py

```diff
@@ -1235,6 +1238,130 @@ def service__get(self):
                     })
             self.print_response(collapsed, format=self.args.format, json=False, table_layout=layout)
 
+    def _get_project_ca(self):
+        return self.client.get_project_ca(project=self.get_project())["certificate"]
+
+    def _get_store_from_args(self):
+        if self.args.overwrite:
+            store = Store.overwrite
+        elif self.args.write:
+            store = Store.skip
+        else:
+            store = Store.skip
+        return store
+
+    @arg.project
+    @arg.service_name
+    @arg("-r", "--route", choices=("dynamic", "privatelink", "public"), default="dynamic")
+    @arg("-a", "--kafka-authentication-method", choices=("certificate", "sasl"), default="certificate")
+    @arg("-u", "--username", default="avnadmin")
+    @arg("--ca", default="ca.pem", dest="ca_path")
+    @arg("--client-cert", default="service.crt", dest="client_cert_path")
+    @arg("--client-key", default="service.key", dest="client_key_path")
+    @arg("-w", "--write", action="store_true", help="Save certificate and key files if they don't not exist")
+    @arg("-ww", "--overwrite", action="store_true", help="Save and overwrite certificate and key files")
```

Comment:
Maybe `W` instead? Multi letter short versions can get quite confusing when you pass multiple parameters with single dash.

Reply:
Changed to `-W`